src/template.html

```diff
@@ -63,16 +63,16 @@
                 </a>
             </button>
             <ul class="dropdown-menu dropdown-menu-right pre-scrollable" aria-labelledby="dropdownMenuButton">
-                <li ng-repeat="repository in getReadableRepositories() | orderBy: ['type === \'system\'', 'id']"
-                    ng-if="repository.id !== getActiveRepository()"
+                <li ng-repeat="repository in getReadableRepositories() | orderBy: ['type === \'system\'', 'id', 'location']"
```

Comment:
Here also I think it will be better to sort by location first and then by id. Perhaps also show the local repos first.